SceneData.js

```diff
@@ -5298,3 +5298,87 @@ var PRESET = {
+        "Attack rolls against the creature have disadvantage, and the creature’s attack rolls have advantage."
+    ],
+    "Paralyzed": [
+        "A paralyzed creature is incapacitated (see the condition) and can’t move or speak.",
+        "The creature automatically fails Strength and Dexterity saving throws.",
+        "Attack rolls against the creature have advantage.",
+        "Any attack that hits the creature is a critical hit if the attacker is within 5 feet of the creature."
+    ],
+    "Petrified": [
+        "A petrified creature is transformed, along with any nonmagical object it is wearing or carrying, into a solid inanimate substance (usually stone). Its weight increases by a factor of ten, and it ceases aging.",
+        "The creature is incapacitated (see the condition), can’t move or speak, and is unaware of its surroundings.",
+        "Attack rolls against the creature have advantage.",
+        "The creature automatically fails Strength and Dexterity saving throws.",
+        "The creature has resistance to all damage.",
+        "The creature is immune to poison and disease, although a poison or disease already in its system is suspended, not neutralized."
+    ],
+    "Poisoned": [
+        "A poisoned creature has disadvantage on attack rolls and ability checks."
+    ],
+    "Prone": [
+        "A prone creature’s only movement option is to crawl, unless it stands up and thereby ends the condition.",
+        "The creature has disadvantage on attack rolls.",
+        "An attack roll against the creature has advantage if the attacker is within 5 feet of the creature. Otherwise, the attack roll has disadvantage."
+    ],
+    "Restrained": [
+        "A restrained creature’s speed becomes 0, and it can’t benefit from any bonus to its speed.",
+        "Attack rolls against the creature have advantage, and the creature’s attack rolls have disadvantage.",
+        "The creature has disadvantage on Dexterity saving throws."
+    ],
+    "Stunned": [
+        "A stunned creature is incapacitated (see the condition), can’t move, and can speak only falteringly.",
+        "The creature automatically fails Strength and Dexterity saving throws.",
+        "Attack rolls against the creature have advantage."
+    ],
+    "Unconscious": [
+        "An unconscious creature is incapacitated (see the condition), can’t move or speak, and is unaware of its surroundings",
+        "The creature drops whatever it’s holding and falls prone.",
+        "The creature automatically fails Strength and Dexterity saving throws.",
+        "Attack rolls against the creature have advantage.",
+        "Any attack that hits the creature is a critical hit if the attacker is within 5 feet of the creature."
+    ]
+}

```

Comment:
I am a bit worried about hard-coding these descriptions in our codebase. 
It would be better if we pulled from an official source, even if it's a one-off async request and we store it globally. But I think it would allow us to avoid having obsolete description in case of errata or updates in the future.

Reply:
Suggestion:

May be out of scope for this PR. We should be able to pull directly from the combat conditions apendix and use that raw. 

https://www.dndbeyond.com/sources/basic-rules/appendix-a-conditions

Reply:
the character sheet has the conditions descriptions, but it needs a button click per condition to reveal the data.

Reply:
Hardcoding is ok from a tech point of view.. I don't think they'll change often (if ever)... What I'm not sure about is if it's legally ok to copy that data inside the plugin .

Reply:
conditions are included in the SRD and are licensed under OGL